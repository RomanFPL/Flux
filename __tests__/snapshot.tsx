import { render } from "@testing-library/react";
// import Home from '@/pages/index'
import Home from "@/app/disabled-slug/page";

it("renders homepage unchanged", () => {
  // const { container } = render(<Home />)
  // expect(container).toMatchSnapshot()
});
