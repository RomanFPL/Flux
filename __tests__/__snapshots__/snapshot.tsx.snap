// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
  <header className={[String]}>
  <Image
    className={[String]}
    src={[String]}
    alt="SkySoftSolutions Logo"
    width={180}
    height={37}
    priority
  />
</header>
<main className={[String]}>
  <h1>This Page is under construction</h1>
</main>
`;
